src/app/components/action-button/index.html

```diff
@@ -31,4 +31,159 @@
+            </p>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-danger">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-inside-long-dialogue-dont.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <header><div class="sky-section-heading">Task placement and behaviors</div></header>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use a set of action buttons inside a modal when no page already exists, or the remaining bulk of the task after the action button decision takes place inside a modal. This is common when creating a new record that only requires a standard modal during different versions of its 'Add' process.
+            </p>
+            <ul>
+                <li>Use the default size for 2 action buttons, and use a large modal for 3 or more.</li>
+                <ul>
+                    <li>Center the action buttons inside the modal and wrap to another row if needed.</li>
+                </ul>
+                <li>When the user selects a choice, dismiss this modal and open whatever container is appropriate for the remainder of the task.</li>
+                <ul>
+                    <li>If users cancel or abandon the task, do not return to the modal.</li>
+                </ul>
+                <li>Always provide a "Cancel" action inside a modal with action buttons.</li>
+            </ul>
+        </stache-column>
+        <stache-column screenSmall="6">
+            <div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/actionbutton/action-buttons-in-modal.png" style="width: 100%;" />
+            </div>
+        </stache-column>
+    </stache-row>
+    
+    <stache-row>
+        <stache-column screenSmall="6">
+            <p>
+            Use action buttons directly on pages or tabs that include add or configuration processes but no other tasks. Most add processes use the modal example above.
+            </p>
+            <ul>
+                <li>If necessary, include a question above the action buttons to explain the decision being made.</li>
+                <li>Center the question and action buttons inside the page.</li>
+                <li>When users select one of the action buttons, replace them with the next step of the task.</li>
+                <li>If users leave the page without selecting an action button, treat the record or process as "incomplete" and place any appropriate status indicators on its parent view.</li>
+                <ul>
+                    <li>An explicit "Cancel" or "Delete" action underneath the action buttons and be used if it is valuable for the user to delete the record entirely from this point.</li>
```

Comment:
I worked on this some more, but it looks like the review conversation was deleted because I made a tweak to this paragraph. What do you think of changing this to:

"If it is useful to include an option for users to back out of configuration processes, place an explicit "Cancel" or "Delete" action under the action buttons."

Reply:
That sounds good.